src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Invoice/AuthorizationTransaction.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace SwedbankPay.Sdk.PaymentInstruments.Invoice
+{
+    public class AuthorizationTransaction : IAuthorizationTransaction
+    {
+        public AuthorizationTransaction(InvoicePaymentAuthorizationDto item)
+        {
+            Addressee = item.Addressee;
+            City = item.City;
+            CoAddress = item.CoAddress;
+            CountryCode = item.CountryCode;
+            CustomerNumber = item.CustomerNumber;
+            Email = new EmailAddress(item.Email);
+            Ip = item.Ip;
+            Msisdn = item.Msisdn;
+            SocialSecurityNumber = item.SocialSecurityNumber;
+            StreetAddress = item.StreetAddress;
+            TransactionActivity = item.TransactionActivity;
+            ZipCode = item.ZipCode;
+        }
+
+        /// <summary>
+        /// The social security number (national identity number) of the consumer. Format Sweden: YYMMDD-NNNN. 
+        /// Format Norway: DDMMYYNNNNN. Format Finland: DDMMYYNNNNN
+        /// </summary>
```

Comment:
These comments should be moved to the `IAuthorizationTransaction.cs` file, so they are exposed to consumers of the SDK.